### PR TITLE
Generate custom attributes of enum fields.

### PR DESCRIPTION
**Pull request by moinessim**
Avoid linebreak between "|" and attributes.
Add linebreak and two spaces after attributes to keep indentation.

### CODE REVIEW

**Comment by drvink** [comment]
@moinessim While I doubt this package has many Mono users, I'd prefer to not break it if possible. Would you be able to investigate why the build is [failing on the Linux test environment](https://travis-ci.org/github/fsprojects/FSharp.Compiler.CodeDom/jobs/745963415)? It does seem to work on OS X, so it might just be an issue with the Mono or F# version used by that version of Ubuntu.

I'm happy to merge this, but if you're not using this package for CodeDom interop or legacy reasons, [FCS is a better choice](https://github.com/fsprojects/FSharp.Compiler.CodeDom/issues/24#issuecomment-366481096) for F# users.

**Comment by moinessim** [comment]
Hi @drvink. Thanks for taking a look at this.
Apparently it's related to fsharp/FAKE#2362.
I tried updating FAKE to the latest version, but I get a scary error from mono.
I am working with CodeDom, generating types (F#) based on XSD.

File: src/FSharp.Compiler.CodeDom/CodeGenerator.fs [hunk 1]
```diff
@@ -1175,7 +1183,7 @@ let getMethodOverloads (membs:CodeTypeMemberCollection) =
 /// fields
 let generateField (c:CodeMemberField) =
   id
-  +> generateCustomAttrDecls c.CustomAttributes ""
+  +> generateCustomAttrDeclsForEnumField c.CustomAttributes ""
```
Comment (moinessim) [inline src/FSharp.Compiler.CodeDom/CodeGenerator.fs]: Oops, this should not change. Confused ``generateField`` with ``generateEnumField``

**Comment by drvink** [comment]
@moinessim Sorry for dragging my feet on this--were you able to figure out the FAKE issue?

**Comment by moinessim** [comment]
> @moinessim Sorry for dragging my feet on this--were you able to figure out the FAKE issue?

Hi @drvink , 
I think to build it with the new version of Fake (recommended fix in issue) would require some changes to the build script, which I'm not familiar with. 
Nevertheless, I was able to build the library with Nix. 
I can make a PR if you're interested.

**Pull request closed without merging**